_conferences/2021/program.html

```diff
@@ -55,5 +135,156 @@ <h4>19 July 2021, 18.30 - 21:00 PM (CEST)</h4>
             <p>During the workshop, we will walk through the Advanced MEI Online Tutorial “Understanding ODD –A tutorial on the internal structure of MEI.” This tutorial introduces the basic concepts of the ODD (One Document Does-it-all) language in which the schema of MEI is written. We will explain the basic principles of the ODD structures found in MEI through the individual examples covered in the tutorial.</p>
         </dd>
     </dl>
-    
+
+    <h3>Paper presentations</h3>
+    <h4 id="oral1">#1. 20 July 2021, 17:45-19:15 (CEST) </h4>
+    <h5>Tools & Applications</h5>
+    <ul>
+        <li>
+            <strong>Codified Spanish music heritage through Verovio: the online platforms Fondo de Música Tradicional IMF-CSIC and Books of Hispanic Polyphony IMF-CSIC</strong>
+            <br>Emilio Ros-Fábregas
+            <br><em>CSIC-Institución Milá y Fontanals of Research in Humanities, Barcelona, Spain</em>
+        </li>
+        <li>
+            <strong>Alleviating the last mile of encoding: The mei-friend package for the Atom text editor</strong>
+            <br>Werner and Goebl David M. Weigl 
+            <br><em>University of Music and Performing Arts Vienna, Austria</em>
+        </li>
+        <li>
+            <strong>Building a Comprehensive Sheet Music Library Application</strong>
+            <br>Klaus Rettinghaus and Alexander Pacha
+            <br><em>Enote GmbH, Germany</em>
+        </li>        
+        <li>
+            <strong>Lohengrin TimeMachine: musicological multimedia made with MELD</strong>
+            <br>David Lewis (1), Kevin Page (1), and Laurence Dreyfus, Laurence (2)            
```

Comment:
Spelling mistake, “Laurence” is repeated

Reply:
Corrected, thanks for spotting it